ml-algorithms/build.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     implementation("ai.djl.onnxruntime:onnxruntime-engine:0.21.0") {
         exclude group: "com.microsoft.onnxruntime", module: "onnxruntime"
     }
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.50"
```

Comment:
Why need to add this dependency?

Reply:
For pulling in kotlin code from [AlertingPluginInterface](https://github.com/opensearch-project/common-utils/blob/main/src/main/kotlin/org/opensearch/commons/alerting/AlertingPluginInterface.kt)

Reply:
Does just using common-utils not provide this?  
```groovy
api "org.opensearch:common-utils:${common_utils_version}@jar"
```

Reply:
@ohltyler Could you please update the response here?